start.py

```diff
@@ -5,26 +5,53 @@
 import threading
 from concurrent import futures
 from os import path
-
+import pkg_resources 
 import grpc
+import numpy as np
+import json
 
 from getgauge import connection, processor
 from getgauge import lsp_server
 from getgauge.impl_loader import copy_skel_files
 from getgauge.messages import lsp_pb2_grpc
 from getgauge.static_loader import load_files
 from getgauge.util import get_step_impl_dir
+from distutils import version
+from subprocess import Popen, PIPE
 
+PLUGIN_JSON = 'python.json'
+VERSION = 'version'
 
 def main():
     _init_logger()
     logging.info("Python: {}".format(platform.python_version()))
     if sys.argv[1] == "--init":
         copy_skel_files()
     else:
-        load_implementations()
-        start()
+        pythonPluginVersion = get_version()
+        getgaugeVersion = version.LooseVersion(pkg_resources.get_distribution('getgauge').version)
+        if not np.array_equal(list(map(int, pythonPluginVersion.split(".")[0:2])), getgaugeVersion.version[0:2]):
```

Comment:
Do we really need to include a new lib (numpy) for comparing a list? I think `==` should work fine.